noir-projects/noir-protocol-circuits/crates/blob/src/blob.nr

```diff
@@ -456,37 +385,6 @@ mod tests {
         assert_eq(lhs, rhs);
     }
 
-    // TODO: After reverting some noir changes (see PR#10341) the below no longer works in unconstrained
-    // This happened previously in commit 893f1eca422d952d672eec75e3c9ff8f149a9ae8 but a sync fixed it.
-    // It works and passes in constrained, just takes a min longer.
-    #[test]
-    fn test_base() {
-        let mut tx_data = FixtureBuilder::new();
-        // Add some random bits of state
-        tx_data.append_note_hashes(50);
-        tx_data.set_protocol_nullifier();
-        tx_data.append_nullifiers(50);
-        tx_data.append_l2_to_l1_msgs(5);
-        tx_data.append_public_logs(5);
-        let mut blob: [Field; FIELDS_PER_BLOB] = [0; FIELDS_PER_BLOB];
-        let blob_fields = tx_data.to_private_to_rollup_accumulated_data().serialize();
-        for i in 0..blob_fields.len() {
-            blob[i] = blob_fields[i];
-        }
-        let mut sponge_blob = SpongeBlob::new(blob_fields.len());
-        sponge_blob.absorb(blob_fields, blob_fields.len());
-
-        let kzg_commitment_in = BlobCommitment { inner: [1, 2] }; // this is made-up nonsense.
-        let padded_blob_fields = pad_end(blob, 0);
-        let hashed_blob = check_block_blob_sponge(padded_blob_fields, sponge_blob);
-        let output = evaluate_blob(blob, kzg_commitment_in, hashed_blob);
-        let expected_z = poseidon::poseidon2::Poseidon2::hash(
-            [sponge_blob.squeeze(), kzg_commitment_in.inner[0], kzg_commitment_in.inner[1]],
-            3,
-        );
-        assert(expected_z == output.z);
-    }
```

Comment:
Removed this test because it did not cover anything not already covered by other single blob tests. The only difference was that it used `FixtureBuilder` and serialised the result, which we don't need to test here.